docs/reference/eql/syntax.asciidoc

```diff
@@ -847,81 +847,165 @@ The query's event items correspond to the following states:
 * State B:  `[process where process.name == "bash"]`
 * Complete: `[process where process.name == "cat"]`
 
+image::images/eql/sequence-state-machine.svg[align="center"]
+
 To find matching sequences, the query uses separate state machines for each
-unique `user.name` value. Pending sequence matches move through each machine's
-states as follows:
+unique `user.name` value. Based on the data set, you can expect two state
+machines: one for the `root` user and one for `elkbee`.
+
+image::images/eql/separate-state-machines.svg[align="center"]
+
+Pending sequence matches move through each machine's states as follows:
 
 [source,txt]
 ----
-{ "index" : { "_id" : "1" } }
+{ "index" : { "_id": "1" } }
 { "user": { "name": "root" }, "process": { "name": "attrib" }, ...}
 // Creates sequence [1] in state A for the "root" user.
 //
-// root: A=[1]
-
-{ "index" : { "_id" : "2" } }
+// +-------------------------root-------------------------+
+// |  +-----------+     +-----------+     +------------+  |
+// |  |  State A  |     |  State B  |     |  Complete  |  |
+// |  +-----------+     +-----------+     +------------+  |
+// |  |    [1]    |     |           |     |            |  |
+// |  +-----------+     +-----------+     +------------+  |
+// +------------------------------------------------------+
```

Comment:
I originally tried using SVG images here similar to the ones above, but they broke up the example too much.